.gitignore

```diff
@@ -25,3 +25,4 @@ build/Release
 # typescript lives in src, output in lib
 lib/
 .vscode
+.idea
```

Comment:
This is the way

Reply:
Tell me you're using the right IDE without telling me you're using the right IDE